src/routes/game/components/dialogs/GameDialogs.vue

```diff
@@ -120,13 +120,8 @@ export default {
     stalemate() {
       return this.gameStore.gameIsOver && this.gameStore.winnerPNum === null;
     },
-    consideringOpponentStalemateRequest: {
-      get() {
-        return this.gameStore.consideringOpponentStalemateRequest;
-      },
-      set(val) {
-        this.gameStore.setConsideringOpponentStalemateRequest(val);
-      },
+    consideringOpponentStalemateRequest() {
+      return this.gameStore.consideringOpponentStalemateRequest;
```

Comment:
Removed setter because the store now fully controls this variable like for the other dialogs, so we only read here. Store sets it based on socket events